src/components/ui/OakRoundIcon/OakRoundIcon.stories.tsx

```diff
@@ -0,0 +1,95 @@
+import React from "react";
+import { Meta, StoryObj } from "@storybook/react";
+
+// @ts-expect-error:  this is an svg
+import lightbulbOutlineIconSvg from "../../../../assets/light-bulb-outline.svg";
+// @ts-expect-error:  this is an svg
+import lightbulbWhiteOutlineSvg from "../../../../assets/light-bulb-with-white-outline.svg";
```

Comment:
These icons have some very particular colour choices which don't lend themselves easily to a monochromatic colour filter. 

I was thinking (at least for now) we could use two separate assets (one with the black stroke and another with the white and yellow fill). Going forward it would be great to be able to apply CSS to icons to target `fill` and `stroke` but that would only be possible with an inline SVG.

Thoughts are very welcome. We could also push back on the multi-colour icon for now?